base/set.jl

```diff
@@ -191,6 +191,62 @@ function unique(f::Callable, C)
 end
 
 """
+    unique!(c)
```

Comment:
Should add `::AbstractVector` since that's the only method for now. Then the description below shouldn't say "collection" : just say `c`. Also, it would be clearer if "as determined..." was right after "duplicate items" (like in `?unique`).